### PR TITLE
Module/location

### PR DESCRIPTION
**Pull request by whatever413y**

### CODE REVIEW

File: modules/location/LocationTypes.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export interface Coordinate {
+    lat: number,
+    long: number
+  }
+  
+export interface Location {
```
Comment (kasutu) [inline modules/location/LocationTypes.ts:6]: add nalang ang `distance` prop di ?

**Review by kasutu — CHANGES_REQUESTED** [review]

File: modules/location/LocationTypes.ts (hunk 1 above)
Comment (whatever413y) [reply modules/location/LocationTypes.ts:6]: if so need ta gid mag add distance for every object sa listing

Comment (kasutu) [reply modules/location/LocationTypes.ts:6]: ara man and distance prop sa line 26 ?

**Review by kasutu — CHANGES_REQUESTED** [review]

File: modules/location/LocationTypes.ts (hunk 1 above)
Comment (whatever413y) [reply modules/location/LocationTypes.ts:6]: additional lang thru map. mas gus2 mo set nalang property?

Comment (kasutu) [reply modules/location/LocationTypes.ts:6]: hmmmm, majo budlay gidman mag butang distance sa location hehe

what if make ta wrapper for `location` called `interface locationWithDistance` as return type ka sorter and `findLocationsNearby`?

**Pull request closed without merging**